app/controllers/endpoints_controller.rb

```diff
@@ -17,9 +17,13 @@ def destroy
     head :no_content
   end
 
+  # rubocop:disable Metrics/MethodLength
   def create
     begin
       endpoint = build_endpoint
+    rescue ActiveRecord::SubclassNotFound => ex
+      render_unprocessable_entity missing_subclass_error(ex)
```

Comment:
I wonder if we can move it to `application_controller` similar to [this](https://github.com/RedHatInsights/notifications-backend/blob/master/app/controllers/application_controller.rb#L9).
If we insist on showing available types, we can wrap the exception with some custom one with the list of available classes.

Personally I don't think this is important enough, since this error should not appear to the enduser.